reflectapi-demo/clients/rust/generated/src/generated.rs

```diff
@@ -1,3 +1,4 @@
+/* <----- */
```

Comment:
is it possible to move this line 3 lines down?

Reply:
good point, I'll just push it after the header which has a few extra lines but no big deal.